app/controlplane/pkg/biz/workflowrun.go

```diff
@@ -42,6 +43,8 @@ type WorkflowRun struct {
 	RunURL, RunnerType    string
 	ContractVersionID     uuid.UUID
 	Attestation           *Attestation
+	Digest                string
```

Comment:
what's this digest for?

Reply:
it holds the digest for either the envelope or the bundle. It's basically the value from the `attestation_digest` field in the database.

Reply:
and why there is another digest in Attestation struct?

Reply:
The one in Attestation is the old one that's only filled when there is a value in the `Attestation` field. I think we can remove that one at this point. Let me review it